packages/@react-spectrum/s2/src/Disclosure.tsx

```diff
@@ -195,7 +200,31 @@ const chevronStyles = style({
   flexShrink: 0
 });
 
+
 function DisclosureHeader(props: DisclosureHeaderProps, ref: DOMRef<HTMLDivElement>) {
+  let {
+    UNSAFE_className,
+    UNSAFE_style,
+    children
+  } = props;
+  let domRef = useDOMRef(ref);
+  return (
+    <div
+      style={UNSAFE_style}
+      className={(UNSAFE_className ?? '') + style({display: 'flex', alignItems: 'center', gap: 4})}
+      ref={domRef}>
+      {children}
+    </div>
+  );
+}
+
+/**
+ * A header for a disclosure. Contains a heading and a trigger button to expand/collapse the panel.
+ */
+let _DisclosureHeader = forwardRef(DisclosureHeader);
+export {_DisclosureHeader as DisclosureHeader};
+
+function DisclosureTrigger(props: DisclosureTriggerProps, ref: DOMRef<HTMLDivElement>) {
   let {
     level = 3,
```

Comment:
I find it a little confusing that the level is defined on the Trigger, not on the Header

Reply:
I just caught up on standup, looks like y'all had some similar misgivings
I'm more in favor of `DisclosureTitle` or something more along those lines